canonical-websites/maas.io
# 2.3 dev with ab

## Note
This is a merge of previously-reviewed 2.3 changes and A/B test variants with a few small copy and image changes from recent reviews.

## Done

Merged in the 2.3 branch and the A/B test branch into one branch and made sure that it was up to date with master.

## QA

Run the site locally and check each page in the main nav has all of the 2.3 changes.
Check that /index2 is also present and up-to-date

## Issue / Card

Fixes #202
